data/human/human missions.txt

```diff
@@ -6301,3 +6301,410 @@ mission "Blind Man from Martini"
+				decline
+			label interested
+			`	You make your way through the crowd to the edge of her personal space bubble. As soon as you do, she extends her hand. "I'm Torrey. Torrey Dupont."`
+			`	You take the hand and shake it, somewhat bemused. "<first> <last>. I'm a -"`
+			`	"An independent captain, yes. I could tell. I would like to hire you to transport a package." Torrey indicates the box beside her with a nudge of her toe.`
+			`	You raise an eyebrow. "Wouldn't it be easier and cheaper to send it via the commercial courier services?"`
+			`	Torrey gives a somewhat sardonic smile. "Well, yes; the thing is, I'm sending it to a friend who's a gunnery sergeant in the militia for the Southern Mutual Defense Pact. Normally, of course, that would be fine, but...then there's who I am." She reaches into a pocket and pulls out a piece of paper. It appears to be a printout of a news headline, "Protests Block Access To Council Chambers For Fourth Straight Day," with a picture under it. Prominent in the picture is a woman holding up a sign decrying recent actions by the government. The woman is unmistakably Torrey.`
+			`	"So you see," she continues, "I'm already being somewhat watched by the Republic's government. What with the rumors lately, I really don't want to do anything that would throw any suspicion onto Irene, but..." she glances down at the box. "She's really been missing some of the stuff from home, and I don't want to make her life less pleasant just because I do things that draw negative attention from high places."`
+			`	She falls silent, waiting impassively for your response.`
+			choice
+				`	"All right; I'll deliver the package."`
+					goto deliver
+				`	"No; I don't support those protests, or the people who participate in them."`
+				`	"No, if you're being watched, then this might get me in trouble too. Sorry."`
+			`	Torrey grimaces, but nods. "That's understandable. Well, have a good day, then." She turns back to looking out at the passing crowds, and you head back on your way.`
+				decline
+			label deliver
+			`	Torrey breaks out into a brilliant grin. "Well, that's great news, then. You'll find Irene at the militia base on <destination> - that's Gunnery Sergeant Irene Brower. I do appreciate this. You'll be paid upon receipt of the package, too, lest you think my appreciation is all I can give."`
+			`	She picks up the package and hands it to you - it's surprisingly heavy, but doesn't seem to shift or rattle at all as you settle it in your arms to carry to your ship.`
+				accept
+	on complete
+		payment 15000 1500
+		conversation
+			`You arrive at the militia base on <destination>, and ask at the gate for GySgt Irene Brower. After a few minutes, a compact young woman with close-cropped blonde hair appears and walks over. When she sees the package at your feet, her face splits into a lopsided grin and her steps quicken. "I see you've brought me a package; this must be from Torrey!"`
```

Comment:
Every military has their own Abbreviation for ranks. An example is for me, my rank is 3SG, and most people wouldn't know what that is. Worst still, is that on my end, MSG is used for the Master Sergeant rank, as compared to the MSGT rank used later in.
I'd advice using full rank names instead, to avoid confusion.

Reply:
To be perfectly honest, I don't remember why I typed that as the abbreviation; it definitely makes more sense to have it written out there.

And, as I noted on Discord, I have no specific attachment to the particular ranks used here for Irene. I just wanted to give a general idea of her position—some form of senior NCO—and the fact that she gets a promotion after the Battle of Greenrock, whether she survives it or not.